bubuku/features/restart_if_dead.py

```diff
@@ -23,9 +24,13 @@ def run(self, current_actions) -> bool:
         if self.broker.is_running_and_registered():
             return False
         _LOG.info('Waiting for complete death')
-        self.broker.stop_kafka_process()
+        if not self.stopped:
+            if not self.broker.stop_kafka_process():
+                return True
```

Comment:
For me it's very confusing that almost every method returns boolean. I prefer exceptions over that. 
I think these methods now behave more like `tryDo` not really `do`